liquibase-couchbase/src/test/java/liquibase/ext/couchbase/mapper/LinesMapperTest.java

```diff
@@ -0,0 +1,61 @@
+package liquibase.ext.couchbase.mapper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Stream;
+
+import liquibase.ext.couchbase.provider.DocumentKeyProvider;
+import liquibase.ext.couchbase.provider.factory.DocumentKeyProviderFactory;
+import liquibase.ext.couchbase.types.DataType;
+import liquibase.ext.couchbase.types.Document;
+import liquibase.ext.couchbase.types.File;
+import liquibase.ext.couchbase.types.Value;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class LinesMapperTest {
+
+    private final DocumentKeyProviderFactory documentKeyProviderFactory = mock(DocumentKeyProviderFactory.class);
+    private final DocumentKeyProvider documentKeyProvider = mock(DocumentKeyProvider.class);
+
+    private final AtomicLong keyHolder = new AtomicLong(1L);
+    private final LinesMapper linesMapper;
+
+    public LinesMapperTest() {
+        linesMapper = new LinesMapper(documentKeyProviderFactory);
+    }
+
+    @BeforeEach
+    public void setUp() {
+        when(documentKeyProviderFactory.getKeyProvider(any())).thenReturn(documentKeyProvider);
+        when(documentKeyProvider.getKey(any())).thenAnswer((args) -> String.valueOf(keyHolder.getAndIncrement()));
+    }
+
+    @Test
+    void Should_map_file_successfully() {
+        File file = mock(File.class);
+        when(file.lines()).thenReturn(Stream.of("{}", "{}", "{}"));
+
+        List<Document> expected = new ArrayList<>();
```

Comment:
I think this could be extracted to the separate method to avoid mixing expected creating logic with test logic

Reply:
Sure, done